drools-benchmarks-parent/README.md

```diff
@@ -0,0 +1,13 @@
+Drools benchmarks
+==========================
+
+The benchmarks are written using [JMH](http://openjdk.java.net/projects/code-tools/jmh/).
+
+This module contains multiple Maven modules. The instructions how to run individual benchmark modules are described in the README files of the modules:
```

Comment:
Can you add some info also for `drools-benchmarks-common`?

Reply:
Fixed.